pascal.js

```diff
@@ -0,0 +1,25 @@
+var row = require("readline");
+var prompts = row.createInterface(process.stdin, process.stdout);
+prompts.question("How many row you want to print ?", function(n) {
+  function factorial(n) {
+    var result = 1;
+
+    for (var c = 1; c <= n; c++) result = result * c;
+
+    return result;
+  }
+
+  var n = 5;
```

Comment:
n is the function argument. So what ever value i pass in there is no use of it.

Reply:
removed var n=5;